### PR TITLE
Added Strong Number program

### PR DESCRIPTION
**Pull request by javeed3157**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given input number is Strong number or not
EX : 1! + 4! + 5! => 1 + 24 + 120 => 145

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

https://github.com/TesseractCoding/NeoAlgo/issues/2826

### CODE REVIEW

**Comment by javeed3157** [comment]
@ankitaggarwal23  I created a new PR and I removed unneccesary spaces I added time and Space complexity . But I am getting Deep Source java(beta) failed .

**Comment by ankitaggarwal23** [comment]
> @ankitaggarwal23 I created a new PR and I removed unneccesary spaces I added time and Space complexity . But I am getting Deep Source java(beta) failed .

You can ignore it, there is some glitch at deepsource's end.

File: Python/math/Strong_Number.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+EX : Consider 145=1! + 4! + 5! = 1 + 24 + 120 = 145
+Here input a number
+We initialize sum to 0 and assign the input number to a temporary variable
+In side while loop --> (we find sum of factorial of each digit of the number)
+In side factorial function --> (we calculate the factorial of each digit and return it)
+In the end if number and the sum obtained is equal then it is a Strong number
+otherwise not
+"""
+
+
```
Comment (ankitaggarwal23) [inline Python/math/Strong_Number.py:13]: Remove extra one line space.

File: Python/math/Strong_Number.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+def factorial(n) :
+    p = 1
+    for i in range(1 , n + 1) :
+        p = p * i
+    return p
+
+
```
Comment (ankitaggarwal23) [inline Python/math/Strong_Number.py:20]: Remove extra one line space.

File: Python/math/Strong_Number.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    if sums == number :
+        print('Number' , number , 'is a Strong Number')
+    else :
+        print('Number' , number , 'is not a Strong Number')
+else :
+    print('0 is not a Strong Number')
+
+# Time Complexity : O(log n)
+# Space Complexity : O(1)
+# TestCase 1:
```
Comment (ankitaggarwal23) [inline Python/math/Strong_Number.py]: Give a new line space after SC.

File: Python/math/Strong_Number.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# Enter a number greater than zero : 65
+# Number 65 is not a Strong Number
+# TestCase 2:
+# Enter a number greater than zero : 145
+# Number 145 is a Strong Number
```
Comment (ankitaggarwal23) [inline Python/math/Strong_Number.py:44]: Give a new line at the EOF.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/math/Strong_Number.py (hunk 1 above)
Comment (javeed3157) [reply Python/math/Strong_Number.py:13]: @ankitaggarwal23 In first commit I gave like that but in DeepSource Python It gave me There must be 2 line spaces btw code and func  I add 1 extra line space btw them

File: Python/math/Strong_Number.py (hunk 2 above)
Comment (javeed3157) [reply Python/math/Strong_Number.py:20]: @ankitaggarwal23 In first commit I gave like that but in DeepSource Python It gave me There must be 2 line spaces btw code and space comp I add 1 extra line space btw them

File: Python/math/Strong_Number.py (hunk 4 above)
Comment (javeed3157) [reply Python/math/Strong_Number.py:44]: @ankitaggarwal23  I already gave that extra line

Comment (javeed3157) [reply Python/math/Strong_Number.py:44]: image
@ankitaggarwal23  I gave it

Comment (javeed3157) [reply Python/math/Strong_Number.py:44]: @ankitaggarwal23  But I dont know why it is not displaying new line here

Comment (ankitaggarwal23) [reply Python/math/Strong_Number.py:44]: Okay then leave I guess there is some issue with that.

Comment (ankitaggarwal23) [reply Python/math/Strong_Number.py:44]: And just fix the remaining changes I marked.

Comment (javeed3157) [reply Python/math/Strong_Number.py:44]: @ankitaggarwal23  I added newline after spacecomplexity

Comment (ankitaggarwal23) [reply Python/math/Strong_Number.py:44]: No I highlighted there are many extra spaces that you have to removed.

**Comment by javeed3157** [comment]
image

@ankitaggarwal23 when I removed extra line space DeepSource Python is saying there must be two line gap thats why I kept 2 line gap now I will commit with  one line gap so that this gets resolved
image

**Comment by javeed3157** [comment]
@ankitaggarwal23  Now it resolved as I commited again by adding extra line space
image

**Review by ankitaggarwal23 — APPROVED** [review]
It okay, I guess there is some glitch in deepsource side.

**Comment by javeed3157** [comment]
@ankitaggarwal23 Thankyou for your patience this is my first Open Source Contribution

**Comment by HarshCasper** [comment]
@all-contributors bot please add @javeed3157 for code

**Pull request merged**